docs/OCP_CI_Tutorials/Scenarios/Scenarios_Guide.md

```diff
@@ -0,0 +1,83 @@
+# (WIP) OpenShift CI Scenarios Guide<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [Overview](#overview)
+- [Provision](#provision)
+- [Orchestrate](#orchestrate)
+- [Execute](#execute)
+- [Report](#report)
+- [Document](#document)
+- [Production](#production)
+  - [Remove Spot Instances](#remove-spot-instances)
+  - [Ensure Trigger is Configured](#ensure-trigger-is-configured)
+  - [Ensure Test execution output matches reporting](#ensure-test-execution-output-matches-reporting)
+  - [Ensure all parties are OWNERS](#ensure-all-parties-are-owners)
+
+
+## Overview
+The following Headers are phases that MUST be done for each scenario. They are needed to provide a structure to the many OpenShift layered product scenarios that will be running on OpenShift CI.
+
+## Provision
+We define provision as the initial setup of your scenarios files and the OpenShift installation
+
+See the [Provision Guide](Provision_Guide.md) for detailed information meant to help you build the foundation of your scenario.
+
+## Orchestrate
+We define Orchestrate as anything, post OpenShift install, that is needed to install and configure your product on OpenShift so that it is ready to be tested.
+
+See the [Orchestration Guide](Orchestration_Guide.md)) for detailed information meant to help you organize the installation of your product.
+
+## Execute
+Execute is defined as the phase where you setup the environment needed for your tests and the step you take to run them.
```

Comment:
```suggestion
Execute is defined as the phase where tests are executed, following any environmental setup in the Orchestrate phase.
```
I feel like "setup the environment needed" could be confused with the orchestrate phase?

Reply:
Fixing this now